internal/util/ctxutil/ctxutil.go

```diff
@@ -61,10 +61,10 @@ func SleepWithJitter(ctx context.Context, d time.Duration, attempts int64) {
 	<-sleepCtx.Done()
 }
 
-// DurationWithJitter returns an exponential backoff duration based on retry with random jitter.
-// The maximum sleep is the cap. The minimum duration is at least 100 milliseconds.
+// DurationWithJitter returns an exponential backoff duration based on retry with random "full jitter".
+// https://aws.amazon.com/blogs/architecture/exponential-backoff-and-jitter/
 //
-// Math/rand is good enough because we don't need the randomness to be cryptographically secure.
+// The maximum sleep is the cap. The minimum duration is at least 100 milliseconds.
```

Comment:
I think 100 milliseconds as the minimum duration is too much. I'd do something like 3-5 milliseconds, and then if it didn't help, we increase it within the next retry.

Reply:
Done